CONTRIBUTING.md

```diff
@@ -11,6 +11,11 @@ You can contribute in the following ways:
 
 If your contributions are accepted into Mastodon, you can request to be paid through [our OpenCollective](https://opencollective.com/mastodon).
 
+Please review the org-level [contribution guidelines] for high-level acceptance
+criteria guidance.
+
+[contribution guidelines]: https://github.com/mastodon/.github/blob/main/CONTRIBUTING.md
```

Comment:
this should probably be an inline link rather than on a separate line, unless I am missing something (does it render differently if linked like this?)

Reply:
It's a way in markdown to keep the paragraphs a little more readable as text -- but still renders as inline links within the paragraph when shown as html. Similar to https://github.com/mastodon/mastodon/blob/main/README.md?plain=1#L153-L158